rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -113,7 +113,7 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
   "evalExpr" should "handle simple addition" in {
     implicit val errorLog = new ErrorLog()
     val result = withTestSpace(errorLog) {
-      case TestFixture(space, reducer) =>
+      case TestFixture(space @ _, reducer) =>
```

Comment:
Please change thos to just `_`. Space is unused directly most of the time, and it's good for the code to tell us that.